projects/js-packages/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This needs to be merged into `projects/js-packages/connection`, we shouldn't have a package.json here.